src/python/pants/backend/python/dependency_inference/module_mapper_test.py

```diff
@@ -36,53 +36,66 @@ def test_default_module_mapping_is_normalized() -> None:
 @pytest.mark.parametrize(
     "stripped_path,expected",
     [
-        (PurePath("top_level.py"), "top_level"),
-        (PurePath("dir", "subdir", "__init__.py"), "dir.subdir"),
-        (PurePath("dir", "subdir", "app.py"), "dir.subdir.app"),
-        (
-            PurePath("src", "python", "project", "not_stripped.py"),
-            "src.python.project.not_stripped",
-        ),
+        ("top_level.py", "top_level"),
+        ("dir/subdir/__init__.py", "dir.subdir"),
+        ("dir/subdir/app.py", "dir.subdir.app"),
+        ("src/python/project/not_stripped.py", "src.python.project.not_stripped"),
     ],
 )
-def test_create_module_from_path(stripped_path: PurePath, expected: str) -> None:
-    assert PythonModule.create_from_stripped_path(stripped_path) == PythonModule(expected)
+def test_create_module_from_path(stripped_path: str, expected: str) -> None:
+    assert PythonModule.create_from_stripped_path(PurePath(stripped_path)) == PythonModule(expected)
 
 
 def test_first_party_modules_mapping() -> None:
     root_addr = Address("", relative_file_path="root.py")
     util_addr = Address("src/python/util", relative_file_path="strutil.py")
+    util_stubs_addr = Address("src/python/util", relative_file_path="strutil.pyi")
```

Comment:
This doesn't reveal a new bug, but I figure is good to make the test more representative of the real world.